cpp/open3d/io/PointCloudIO.cpp

```diff
@@ -51,6 +51,15 @@ static const std::unordered_map<
                 {"pts", ReadPointCloudFromPTS},
         };
 
+static const std::unordered_map<
+        std::string,
+        std::function<bool(const std::vector<char> &,
```

Comment:
I would recommend `const unsigned char*` rather than `const std::vector<char>&`. Raw buffers are typically more flexible in lower-level APIs. With raw buffers, we will need to specify the buffer size as an input.

Reply:
Done in 1984a20